shell/platform/android/io/flutter/plugin/common/PluginRegistry.java

```diff
@@ -80,6 +80,13 @@
          */
         Context context();
 
+        /**
+         * Returns the active {@link Context}.
```

Comment:
We should document exactly what is returned, e.g.
```java
/**
 * Returns the active {@link Context}.
 *
 * @return the current {@link #activity() Activity}, if not null, otherwise the {@link #context() Application}.
 */
```

Reply:
Updated